--- etc/xdg/autostart/user-dirs-update-gtk.desktop	2023-12-22 15:02:35.687540300 +0100
+++ etc/xdg/autostart/user-dirs-update-gtk.desktop	2023-12-22 15:03:33.507101812 +0100
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Encoding=UTF-8
-Exec=xdg-user-dirs-gtk-update
+Exec=env LC_MESSAGES=en_US xdg-user-dirs-gtk-update
 Name=User folders update
 Name[an]=Esvielle d'as carpetas de l'usuario
 Name[ar]=تحديث مجلدات المستخدم
